src/main/resources/sql/maxwell_schema.sql

```diff
@@ -1,34 +1,58 @@
-CREATE DATABASE IF NOT EXISTS `maxwell`;
+SET @c = CONCAT('CREATE DATABASE IF NOT EXISTS ', @maxwell_db)
 
-CREATE TABLE IF NOT EXISTS `maxwell`.`schemas` (
+PREPARE maxwell_stmt from @c
+
+EXECUTE maxwell_stmt
+
+DEALLOCATE PREPARE maxwell_stmt
+
+SET @c = CONCAT('CREATE TABLE IF NOT EXISTS `', @maxwell_db, '`.`schemas` (
```

Comment:
I think the same solution should stand here:  have the calling code perform the `create database if not exists` and then execute the sql file under a "USE db" style connection.